docs/integrations/prometheus.txt

```diff
@@ -37,10 +37,13 @@ file.
 
 The Determined Prometheus endpoint is configured to work with `cAdvisor
 <https://github.com/google/cadvisor>`__ for CPU metrics and `dcgm-exporter
-<https://github.com/NVIDIA/dcgm-exporter>`__ (DCGM) for GPU metrics. Although other monitoring tools
-can be used with the setup, this guide describes only cAdvisor and DCGM tool configuration.
-Prometheus queries on metrics collected by other tools can differ, depending on the format and
-organization of the returned metrics.
+<https://github.com/NVIDIA/dcgm-exporter>`__ (DCGM) for GPU metrics. The default ports for cAdvisor
+and dcgm-exporter (8080 and 9400, respectively) are hardcoded into the Prometheus endpoint. If an
```

Comment:
Change "The default ports for cAdvisor and dcgm-exporter (8080 and 9400, respectively) are ..."
to: "The default ports for cAdvisor, 8080, and dcgm-exporter, 9400, are ..."
Parentheses are not permitted. "Respectively" is too wordy.